server/api/router.go

```diff
@@ -172,13 +175,13 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 
 	confHandler := newConfHandler(svr, rd)
 	registerFunc(apiRouter, "GetConfig", "/config", confHandler.Get, setMethods("GET"))
-	registerFunc(apiRouter, "SetConfig", "/config", confHandler.Post, setMethods("POST"))
+	registerFunc(apiRouter, "SetConfig", "/config", confHandler.Post, setMethods("POST"), setAuditBackend(localLog))
```

Comment:
Maybe we need to improve the readability later. it is not easy to see which APIs are set audited from the code.